### PR TITLE
Story-871- Finos prep

### PR DESCRIPTION
**Pull request by PayalKhanna**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Code of Conduct for Rune
```
Comment (ja6a-regnosys) [inline CODE_OF_CONDUCT.md:1]: Rune -> Rosetta Testing?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,141 +1,55 @@
-# How to contribute #
+# Rune Contribution and Governance Policies
```
Comment (ja6a-regnosys) [inline CONTRIBUTING.md:1]: Rune -> Rosetta Testing?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,141 +1,55 @@
...
 
-We'd love for you to contribute to rosetta-testing. There are
-a just a few small guidelines  we would like you to follow:
+This document describes the contribution process and governance policies of the FINOS Rune project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy]( https://community.finos.org/governance-docs/IP-policy.pdf), [Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct), [Collaborative Principles](https://community.finos.org/docs/governance/collaborative-principles/), and [Meeting Procedures](https://community.finos.org/docs/governance/meeting-procedures/).
```
Comment (ja6a-regnosys) [inline CONTRIBUTING.md:3]: Rune -> Rosetta Testing?

File: README.md [hunk 4]
```diff
@@ -1,35 +1,62 @@
...
+<dependency>
+    <groupId>com.regnosys</groupId>
+    <artifactId>rosetta-testing</artifactId>
+    <version>0.0.0-SNAPSHOT</version>
+</dependency>
 ```
 
-## How to contribute
+## Contributing
+For any questions, bugs or feature requests please open an [issue](https://github.com/finos/rune-dsl/issues)
```
Comment (ja6a-regnosys) [inline README.md:34]: wrong URL

File: README.md [hunk 5]
```diff
@@ -1,35 +1,62 @@
...
+For anything else please send an email to {project mailing list}.
 
-Please read the [detailed guide](https://github.com/REGnosys/rosetta-testing/blob/master/CONTRIBUTING.md).
+To submit a contribution:
+1. Fork it (<https://github.com/finos/rune-testing/fork>)
```
Comment (ja6a-regnosys) [inline README.md:38]: Rune -> Rosetta Testing?

File: README.md [hunk 6]
```diff
@@ -1,35 +1,62 @@
...
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
+
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool (or [EasyCLA](https://community.finos.org/docs/governance/Software-Projects/easycla)). Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
+
+* Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+
+## Get in touch with the Rune Team
```
Comment (ja6a-regnosys) [inline README.md:50]: Rune -> Rosetta Testing?

File: README.md [hunk 7]
```diff
@@ -1,35 +1,62 @@
...
+
+Get in touch with the Rune team by creating a [GitHub issue](https://github.com/REGnosys/rosetta-testing/issues/new) and labelling it with "help wanted".
```
Comment (ja6a-regnosys) [inline README.md:52]: Rune -> Rosetta Testing?

File: README.md (hunk 5 above)
Comment (ja6a-regnosys) [reply README.md:38]: wrong url

File: pom.xml [hunk 8]
```diff
@@ -10,7 +30,7 @@
         <version>5</version>
     </parent>
 
-    <name>rosetta-testing</name>
+    <name>Rosetta Testing</name>
```
Comment (ja6a-regnosys) [inline pom.xml:33]: revert this change i.e. use the old name

Comment (ja6a-regnosys) [reply pom.xml:33]: that goes for rosetta-common too

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 9]
```diff
@@ -495,6 +515,38 @@
...
+                    <inceptionYear>2022</inceptionYear>
+                    <roots>
+                        <root>src/main/java</root>
+                        <root>src/test</root>
+                    </roots>
+                    <includes>
+                        <include>**/*.java</include>
+                        <include>**/*.xml</include>
+                    </includes>
+                    <processStartTag> ==============</processStartTag>
```
Comment (ja6a-regnosys) [inline pom.xml:536]: can u add one more character please.

File: src/main/java/com/regnosys/testing/CompiledCode.java [hunk 10]
```diff
@@ -1,5 +1,25 @@
 package com.regnosys.testing;
 
+/*-
+ * ==============
```
Comment (ja6a-regnosys) [inline src/main/java/com/regnosys/testing/CompiledCode.java:4]: if u add one more character then this will line up with proceeding text

File: CONTRIBUTING.md
Comment (ja6a-regnosys) [inline CONTRIBUTING.md:1]: is this file CONTRIBUTING.md pretty much verbatim with the finos blueprint example?

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (PayalKhanna) [reply CONTRIBUTING.md:1]: yes this was taken from their templates

File: pom.xml [hunk 11]
```diff
@@ -495,6 +515,38 @@
...
+                    <sectionDelimiter>===============</sectionDelimiter>
+                    <processEndTag>   ==============</processEndTag>
```
Comment (ja6a-regnosys) [inline pom.xml:538]: missing

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 12]
```diff
@@ -495,6 +515,38 @@
...
+                    <roots>
+                        <root>src/main/java</root>
+                        <root>src/test</root>
+                    </roots>
+                    <includes>
+                        <include>**/*.java</include>
+                        <include>**/*.xml</include>
+                    </includes>
+                    <processStartTag> ==============</processStartTag>
+                    <sectionDelimiter>===============</sectionDelimiter>
```
Comment (ja6a-regnosys) [inline pom.xml:537]: one too many

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 12 above)
Comment (ja6a-regnosys) [reply pom.xml:537]: or one too little on the other two - i think it is the latter.

**Review by ja6a-regnosys — APPROVED** [review]
